ext/node/polyfills/worker_threads.ts

```diff
@@ -53,7 +53,17 @@ class _Worker extends EventEmitter {
       specifier = `data:text/javascript,${specifier}`;
     } else if (typeof specifier === "string") {
       specifier = resolve(specifier);
-      if (!specifier.toString().endsWith(".mjs")) {
+      let pkg;
+      try {
+        pkg = Deno[Deno.internal].core.ops
+          .op_require_read_closest_package_json(specifier);
```

Comment:
Please import `core` at the top of the file like so:
```
const { core } = globalThis.__bootstrap;
```
This way we don't have to rely on `Deno[Deno.internal]` which will be removed in the near future.